cpp/src/io/parquet/page_string_decode.cu

```diff
@@ -1076,7 +1076,7 @@ CUDF_KERNEL void __launch_bounds__(decode_block_size)
             __syncwarp();
           } else if (use_char_ll) {
             __shared__ __align__(8) uint8_t const* pointers[warp_size];
-            __shared__ __align__(4) size_type offsets[warp_size];
+            __shared__ __align__(4) size_t offsets[warp_size];
```

Comment:
```suggestion
            __shared__ __align__(8) size_t offsets[warp_size];
```
Are these align declarators needed?

Reply:
tbh, I started using the `__align__` as a monkey-see-monkey-do kind of thing :sweat_smile:. I don't know if they're actually necessary at this point.

Reply:
I suspect this one is the only one that may have purpose
https://github.com/rapidsai/cudf/pull/15632/files#diff-52e09ddca44181e11af56d8526360207906f5f25ba888cf51efbd2c1b15d775cR957
```
__shared__ __align__(16) page_state_s state_g;
```
Only because `page_state_s` is a structure but it should probably have been declared with `alignas(16)`

Reply:
I'll remove them since they're unnecessary.